tutorial/markdown/aspnet/quickstart-csharp-aspnet-minapi/quickstart-csharp-aspnet.md

```diff
@@ -19,383 +21,627 @@ sdk_language:
 
-<!--
+## Prerequisites
+To run this prebuilt project, you will need:
 
-**remove gitpod until they fix it**
+- [Couchbase Capella](https://www.couchbase.com/products/capella/) cluster with [travel-sample](https://docs.couchbase.com/dotnet-sdk/current/ref/travel-app-data-model.html) bucket loaded.
+    - To run this tutorial using a self managed Couchbase cluster, please refer to the [appendix](#running-self-managed-couchbase-cluster).
+- [.NET SDK v6+](https://dotnet.microsoft.com/en-us/download/dotnet) installed.
+    - Ensure that the .Net version is [compatible](https://docs.couchbase.com/dotnet-sdk/current/project-docs/compatibility.html#dotnet-compatibility) with the Couchbase SDK.
+- Code Editor installed (Visual Studio Professional, Visual Studio Code, or JetBrains Rider)
+- Loading Travel Sample Bucket
+    - If travel-sample is not loaded in your Capella cluster, you can load it by following the instructions for your Capella Cluster:
+        - [Load travel-sample bucket in Couchbase Capella](https://docs.couchbase.com/cloud/clusters/data-service/import-data-documents.html#import-sample-data)
 
-[![Try it now!](https://da-demo-images.s3.amazonaws.com/runItNow_outline.png?couchbase-example=aspnet-quickstart-repo&source=github)](https://gitpod.io/#https://github.com/couchbase-examples/aspnet-quickstart)
+### Couchbase Capella Configuration
 
--->
+When running Couchbase using [Capella](https://cloud.couchbase.com/), the following prerequisites need to be met.
 
-## Prerequisites
+- The application requires the travel-sample bucket to be [loaded](https://docs.couchbase.com/cloud/clusters/data-service/import-data-documents.html#import-sample-data) in the cluster from the Capella UI.
+- Create the [database credentials](https://docs.couchbase.com/cloud/clusters/manage-database-users.html) to access the travel-sample bucket (Read and Write) used in the application.
+- [Allow access](https://docs.couchbase.com/cloud/clusters/allow-ip-address.html) to the Cluster from the IP on which the application is running.
 
-To run this prebuilt project, you will need:
+## App Setup
 
-* Follow [Couchbase Installation Options](/tutorial-couchbase-installation-options) for installing the lastest Couchbase Database Server Instance
-* [.NET SDK v6](https://dotnet.microsoft.com/download/dotnet/6.0) installed
-* Basic knowledge of [ASP.NET Minimum API](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/minimal-apis?view=aspnetcore-6.0) 
-* Code Editor installed (Visual Studio Professional, Visual Studio for Mac, or Visual Studio Code)
+We will walk through the different steps required to get the application running.
 
-## Clone The Project
+### Cloning Repo
 
 ```shell
-git clone https://github.com/couchbase-examples/aspnet-quickstart-minapi
+git clone https://github.com/couchbase-examples/aspnet-minapi-quickstart-travelsample.git
```

Comment:
Please double-check the URL. It looks wrong.

Reply:
Changes done.